addon/templates/components/polaris-form-layout/group.hbs

```diff
@@ -9,3 +13,7 @@
     {{text}}
   {{/if}}
 </div>
+
+{{#if helpText}}
+  <div id={{helpTextID}} class="Polaris-FormLayout__HelpText" data-test-form-layout-group-helpText>{{render-content helpText}}</div>
```

Comment:
nit: Do we want to break up the element blocks here for easier reading?

```hbs
<div id={{helpTextID}} class="Polaris-FormLayout__HelpText" data-test-form-layout-group-helpText>
  {{render-content helpText}}
</div>
```

If so, same with the title. But just a suggestion.